sdk/nodejs/resource.ts

```diff
@@ -269,6 +269,13 @@ export abstract class CustomResource extends Resource {
     // tslint:disable-next-line:variable-name
     public readonly __pulumiCustomResource: boolean;
 
+    /**
+     * Private field containing the type ID for this object. Useful for implementing `isInstance` on
+     * classes that inherit from `CustomResource`.
+     */
+    // tslint:disable-next-line:variable-name
+    /* @internal */ public readonly __pulumiType: string;
```

Comment:
Cyrus recently did a pass here where he made sure that `@internal` was inside a doccomment style comment (e.g. `/** `) because otherwise I think these members were not being elided from our docs.  Can you move the `@internal` decorator to the top of comment?